packages/core-components/package.json

```diff
@@ -67,6 +67,7 @@
     "react-virtualized-auto-sizer": "^1.0.6",
     "react-window": "^1.8.6",
     "remark-gfm": "^3.0.1",
+    "unist-util-position": "^4.0.4",
```

Comment:
Was this necessary?

Reply:
I didn't see any other file in core-components being changed, just this package added.

Reply:
@aitherios it was a tentative of fixing this build error, but I removed this since it does not fix the problem:
https://github.com/backstage/backstage/actions/runs/3985180683/jobs/6832242810
@backstage/maintainers do you have any recommendation?

Reply:
It's a breakage in a transitive upstream dependency. Looks like they're on it though so hopefully fixed soon https://github.com/syntax-tree/unist-util-position/issues/10

Reply:
@Rugvip How does this fix sound to you?
https://github.com/backstage/backstage/pull/15855/commits/136e333f6ce77e3993a751e3e864447f361175a1

Reply:
Last commit removed, it is working because a new release is out (Thanks @Rugvip)